content/learn/node-request-data.md

```diff
@@ -23,26 +21,23 @@ axios.post('https://whatever.com/todos', {
 This is the matching server-side code:
 
 ```js
-const express = require('express');
-
-const app = express();
-
-app.use(
-  express.urlencoded({
-    extended: true,
-  })
-);
-
-app.use(express.json());
+const http = require('node:http');
 
-app.post('/todos', (req, res) => {
-  console.log(req.body.todo);
+const server = http.createServer((req, res) => {
+  let data = '';
+  req.on('data', chunk => {
+    data += chunk;
+  });
+  req.on('end', () => {
+    console.log(JSON.parse(data).todo); // 'Buy the milk'
+    res.end();
+  });
 });
-```
 
-If you're not using Express and you want to do this in vanilla Node.js, you need to do a bit more work, of course, as Express abstracts a lot of this for you.
+server.listen(3000);
+```
 
-The key thing to understand is that when you initialize the HTTP server using `http.createServer()`, the callback is called when the server gets all the HTTP headers, but not the request body.
+You must understand that when you initialize the HTTP server using `http.createServer()`, the callback is called when the server gets all the HTTP headers, but not the request body.
```

Comment:
Honestly speaking, I think the tone of `The key thing to understand is` suits better and doesn't sound as imperative as `You must / We must`